src/api/Jalasoft.TeamUp.Resumes.API/Controllers/GetSkillsByName.cs

```diff
@@ -23,8 +22,8 @@ public GetSkillsByName(ISkillsService skillsService)
 
         [FunctionName("GetSkillsByName")]
         [OpenApiOperation(operationId: "GetSkillsByName", tags: new[] { "Skills" })]
-        [OpenApiParameter(name: "name", In = ParameterLocation.Query, Required = false, Type = typeof(string), Description = "The name of the skill to search by.")]
-        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Skill), Description = "Successful response")]
+        [OpenApiParameter(name: "name", In = ParameterLocation.Query, Required = true, Type = typeof(string), Description = "The name of the skill to search by.")]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Skill[]), Description = "Successful response")]
         [OpenApiResponseWithoutBody(statusCode: HttpStatusCode.NotFound, Description = "Resource not found")]
 
         public IActionResult Run(
```

Comment:
in line 34 you are returning a successful case only but not a NotFound that you mention in line 27

Reply:
please return a 404 not found case

Reply:
reactivating this:
any updates about this? where are you handling the 404?